Cargo.toml

```diff
@@ -5,7 +5,13 @@ license = "MIT"
 authors = ["Nervos Core Dev <dev@nervos.org>"]
 edition = "2018"
 build = "build.rs"
-description = "TODO(doc): @doitian crate description"
+description = """
+TODO(doc): @doitian crate description
+
+## Minimum Rust version policy (MSRV)
```

Comment:
It seems that docs.rs don't recognize markdown syntax, for example [here](https://docs.rs/crate/escposify/0.4.0).

Reply:
Any suggestions?

Reply:
Generate `README.md` for each crate, write licenses, MSRV, descriptions and any things which can generated from current source codes into it, then link it to the main `README.md`?
p.s. The script to generate `README.md` should be added into CI.

Reply:
I think the README.md can be generated before `cargo publish`, since we already enabled `--alow-dirty` to disable dev dependencies.